contributing/component-validator-rules.md

```diff
@@ -385,12 +385,115 @@ attrs: {
 }
 ```
 
+## Test Files
+
+It is a good idea to contribute test files along with your validator rules
+which at minimum demonstrate a correct usage of your validator rules.
+
+A good place to start is to copy
+[minimum_valid_amp.html](https://github.com/ampproject/amphtml/blob/master/validator/testdata/feature_tests/minimum_valid_amp.html) to a new file named, for example:
+
+<pre>
+amphtml/extensions/<b>amp-cat</b>/0.1/test/validator-<b>amp-cat</b>.html
+</pre>
+
+This file looks like:
```

Comment:
Why copy it into here? I think a link is better so if it's updated this doc will still be fine.